src/index.js

```diff
@@ -11,18 +13,20 @@ form.addEventListener('submit', (e) => {
 
```

Comment:
I just realised this is just a built-in class!!
spent a good while understanding what this is and how it works.
well done!

ref: https://developer.mozilla.org/en-US/docs/Web/API/FormData